pkg/defaults/defaults.go

```diff
@@ -415,17 +416,17 @@ func leasesForTest(s *api.MultiStageTestConfigurationLiteral) (ret []api.StepLea
 // the root image.
 func rootImageResolver(client loggingclient.LoggingClient, ctx context.Context) func(root, cache *api.ImageStreamTagReference) (*api.ImageStreamTagReference, error) {
 	return func(root, cache *api.ImageStreamTagReference) (*api.ImageStreamTagReference, error) {
-		log.Printf("Determining if build cache %s can be used in place of root %s\n", cache.ISTagName(), root.ISTagName())
+		logrus.Debugf("Determining if build cache %s can be used in place of root %s\n", cache.ISTagName(), root.ISTagName())
```

Comment:
Do we already have a way to enable debug level?

Reply:
Yes. Debug and trace go into `$ARTIFACT_DIR/ci-operator.log` instead of stdout.